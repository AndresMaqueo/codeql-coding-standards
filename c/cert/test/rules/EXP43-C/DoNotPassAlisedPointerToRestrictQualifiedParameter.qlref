rules/EXP43-C/DoNotPassAlisedPointerToRestrictQualifiedParameter.ql