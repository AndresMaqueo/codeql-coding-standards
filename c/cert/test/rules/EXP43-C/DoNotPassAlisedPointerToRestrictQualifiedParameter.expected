| test.c:53:3:53:6 | call to copy | Call to 'copy' passes an aliased pointer to a restrict-qualified parameter. |
| test.c:58:3:58:6 | call to copy | Call to 'copy' passes an aliased pointer to a restrict-qualified parameter. |
| test.c:64:3:64:8 | call to strcpy | Call to 'strcpy' passes an aliased pointer to a restrict-qualified parameter. |
| test.c:71:3:71:8 | call to memcpy | Call to 'memcpy' passes an aliased pointer to a restrict-qualified parameter. |
| test.c:84:3:84:7 | call to scanf | Call to 'scanf' passes an aliased pointer to a restrict-qualified parameter. |
